### ISSUE TITLE
AsciiBib crashes on link string

### ISSUE THREAD

**Issue by manuelfuenmayor** [relaton/relaton-bib#22]
I did a test of AsciiBib using the following sample:
````
[[OGC-14-005r5]]
[%bibitem]
=== OGC Indoor Geography Markup Language (IndoorGML)
id:: 14-005r5
docid::
  type::: OGC
  id::: OGC 14-005r5
link:: http://docs.opengeospatial.org/is/14-005r5/14-005r5.html
date::
  type::: published
  value::: 2018
````
And, it didn't compile at first. 

*Error log:*

````
$ bundle exec metanorma -t ogc -x html sources/19-004/document.adoc
NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
Gem::Specification#rubyforge_project= called from C:/tools/ruby25/lib/ruby/gems/2.5.0/specifications/sterile-1.0.14.gemspec:15.
asciidoctor: ERROR: sources/19-004/sections/03-references.adoc: line 34: bibliography sections do not support nested sections
C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/relaton-bib-0.8.1/lib/relaton_bib/bibliographic_item.rb:507:in `block (2 levels) in render_xml': undefined method `to_xml' for "http://docs.opengeospatial.org/is/14-005r5/14-005r5.html":String (NoMethodError)
Did you mean?  to_yaml
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/relaton-bib-0.8.1/lib/relaton_bib/bibliographic_item.rb:507:in `each'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/relaton-bib-0.8.1/lib/relaton_bib/bibliographic_item.rb:507:in `block in render_xml'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/nokogiri-1.10.9-x64-mingw32/lib/nokogiri/xml/builder.rb:394:in `insert'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/nokogiri-1.10.9-x64-mingw32/lib/nokogiri/xml/builder.rb:377:in `method_missing'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/relaton-bib-0.8.1/lib/relaton_bib/bibliographic_item.rb:503:in `render_xml'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/relaton-bib-0.8.1/lib/relaton_bib/bibliographic_item.rb:277:in `block in to_xml'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/nokogiri-1.10.9-x64-mingw32/lib/nokogiri/xml/builder.rb:295:in `initialize'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/relaton-bib-0.8.1/lib/relaton_bib/bibliographic_item.rb:276:in `new'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/relaton-bib-0.8.1/lib/relaton_bib/bibliographic_item.rb:276:in `to_xml'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.3.21/lib/asciidoctor/standoc/cleanup_ref.rb:121:in `block in ref_dl_cleanup'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/nokogiri-1.10.9-x64-mingw32/lib/nokogiri/xml/node_set.rb:238:in `block in each'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/nokogiri-1.10.9-x64-mingw32/lib/nokogiri/xml/node_set.rb:237:in `upto'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/nokogiri-1.10.9-x64-mingw32/lib/nokogiri/xml/node_set.rb:237:in `each'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.3.21/lib/asciidoctor/standoc/cleanup_ref.rb:118:in `ref_dl_cleanup'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.3.21/lib/asciidoctor/standoc/cleanup.rb:42:in `cleanup'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-ogc-1.0.3/lib/asciidoctor/ogc/converter.rb:124:in `cleanup'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.3.21/lib/asciidoctor/standoc/base.rb:178:in `makexml'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-ogc-1.0.3/lib/asciidoctor/ogc/converter.rb:24:in `makexml'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-ogc-1.0.3/lib/asciidoctor/ogc/converter.rb:55:in `document'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.3.21/lib/asciidoctor/standoc/utils.rb:116:in `convert'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/document.rb:951:in `convert'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/convert.rb:120:in `convert'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-1.0.0/lib/metanorma/input/asciidoc.rb:10:in `process'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-ogc-1.0.3/lib/metanorma/ogc/processor.rb:26:in `input_to_isodoc'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-1.0.0/lib/metanorma/compile.rb:112:in `process_input'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-1.0.0/lib/metanorma/compile.rb:17:in `compile'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-cli-1.2.10.2/lib/metanorma/cli/compiler.rb:41:in `compile_file'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-cli-1.2.10.2/lib/metanorma/cli/compiler.rb:29:in `compile'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-cli-1.2.10.2/lib/metanorma/cli/compiler.rb:33:in `compile'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-cli-1.2.10.2/lib/metanorma/cli/command.rb:35:in `compile'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-cli-1.2.10.2/lib/metanorma/cli.rb:60:in `start'
        from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/metanorma-cli-1.2.10.2/exe/metanorma:20:in `<top (required)>'
        from C:/tools/ruby25/bin/metanorma:23:in `load'
        from C:/tools/ruby25/bin/metanorma:23:in `<main>'
````
Apparently, there is an error related with the line: 

`link:: http://docs.opengeospatial.org/is/14-005r5/14-005r5.html`

So, after removing that, I was able to compile.

Thanks!

**Comment by ronaldtse** [comment]
Thanks @manuel489 . @andrew2net can you help fix this? We got a report that AsciiBib doesn't work and we need to fix it.

**Comment by andrew2net** [comment]
Fixed it, please update relaton-bib to v0.9.1

**Comment by ronaldtse** [comment]
Thanks @andrew2net !
